DEBUG.md

```diff
@@ -23,7 +24,7 @@ The `mock-client` directory contains debug clients for testing the Socket MCP se
 Direct stdio communication using JSON-RPC protocol:
 
 ```bash
-npm run debug:stdio
+npm run debug-stdio
```

Comment:
I switched to `-` instead of `:` to disambiguate npm scripts that are run with globs and those just grouped together that don't run together.